docs/src/api/class-route.md

```diff
@@ -49,7 +49,7 @@ await page.route('**/*', (route, request) => {
     foo: 'foo-value', // set "foo" header
     bar: undefined, // remove "bar" header
   };
-  route.continue({headers});
+  route.continue({headers:{...headers});
```

Comment:
I believe the only difference is that your code creates a copy of the headers map. Since we don't modify I'd leave the more concise version.